MarkBind/markbind
# Update seamless panels to inherit background color from parent

**What is the purpose of this pull request?**

ReaderUsability


Resolves #1697 

**Overview of changes:**
Updated CSS for child components to inherit background color of parent component.

_Before_
<img width="988" alt="image" src="https://user-images.githubusercontent.com/62977276/177548802-5d859768-4c9c-450f-aae7-d2ca8a5522a6.png">

_After_
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/62977276/177548681-2ba3639e-0bf0-4bca-8ca1-67585e0cd23e.png">

**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**
Check the User Guide --> Authoring Contents --> Presentation --> Panels for how the different panels would look like.
Confirm that the seamless panels inherit the colour of their parent container.
Check Home --> More control to the reader, without duplicating code. The nested panel there should inherit the box colour.



**Proposed commit message: (wrap lines at 72 characters)**
Update CSS for seamless panels to inherit parent's background color


---

**Checklist:** :ballot_box_with_check:
